components/script/script_thread.rs

```diff
@@ -258,7 +261,7 @@ impl InProgressLoad {
             is_visible: true,
             url: url,
             origin: origin,
-            navigation_start: (current_time.sec * 1000 + current_time.nsec as i64 / 1000000) as u64,
+            navigation_start: navigation_start,
```

Comment:
```suggestion
            navigation_start,
```

Reply:
I contemplated doing this but it seems this would be better dealt with in a big `fmt` change.

Reply:
We could enforce it with [use_field_init_shorthand](https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=use_field_init_shorthand#use_field_init_shorthand).